CRM/Import/Form/DataSource.php

```diff
@@ -32,6 +32,30 @@ public function preProcess() {
 
     // check for post max size
     CRM_Utils_Number::formatUnitSize(ini_get('post_max_size'), TRUE);
+    $this->assign('importEntity', $this->getTranslatedEntity());
+    $this->assign('importEntities', $this->getTranslatedEntities());
+  }
+
+  /**
+   * Get the import entity (translated).
+   *
+   * Used for template layer text.
+   *
+   * @return string
+   */
+  protected function getTranslatedEntity(): string {
+    return '';
```

Comment:
@eileenmcnaughton your approach to strings looks good, but you could actually simplify it further and not require each child class to override the `getTranslatedEntity/Entities` functions. This ought to work:

```suggestion
    return Civi\Api4\Utils\CoreUtil::getInfoItem($this::IMPORT_ENTITY, 'title');
```

And the other one can do the same thing using `'title_plural'`.

Reply:
@colemanw that works - it does change the capitalisation - but I think that is acceptable

![image](https://user-images.githubusercontent.com/336308/163465881-e8e365ce-c50d-4bd1-8ea7-a565274bc0dd.png)